test/Parse/recovery.swift

```diff
@@ -459,9 +459,18 @@ func use_BracesInsideNominalDecl1() {
   var _ : BracesInsideNominalDecl1.A // no-error
 }
 
+class SR771 { // expected-note {{in declaration of 'SR771':}}
+    print("No one else was in the room where it happened") // expected-error {{expected declaration}}
+}
+
+extension SR771 { // expected-note {{in extension of 'SR771':}}
+    print("The room where it happened, the room where it happened") // expected-error {{expected declaration}}
```

Comment:
I'm still trying to sneak this one in:
![372](https://cloud.githubusercontent.com/assets/75067/13186026/ec0731d8-d6f8-11e5-9d19-93865613692a.jpg)

😀